libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -624,12 +583,11 @@ class SolanaWeb3Manager {
 
   /**
    * Fetch account on Solana given the program derived address
-   * @param {PublicKey} pda the program derived address from findDerivedPair()
    */
-  async fetchAccount (pda) {
+  async fetchAccount(pda: Address) {
```

Comment:
it's not a PublicKey? Did the old JS comment lie?

Reply:
yeah that works too, interesting, so Address is string | PublicKey, just went with PublicKey